packages/osbx/osbx.1.0.0/opam

```diff
@@ -18,3 +18,4 @@ depends: [
   "ctypes-foreign" {build}
   "cmdliner" {build}
 ]
+available: [ ocaml-version >= "4.03.0" ]
```

Comment:
Had to add this because 4.02.3 builds were failing with a syntax error, which you can see [here](https://travis-ci.org/ocaml/opam-repository/jobs/287979507).